search-app/src/cmr/search/services/query_walkers/update_field_resolver.clj

```diff
@@ -0,0 +1,95 @@
+(ns cmr.search.services.query-walkers.update-field-resolver
```

Comment:
Yeah, I took a look at common-app-lib services unit tests and the search-app services unit tests; looks like adding these would be fairly straight-forward.